src/Blazor.BrowserExtension.Build/build/Blazor.BrowserExtension.Build.targets

```diff
@@ -82,6 +82,10 @@
         <From>this.mono_wasm_assembly_already_added</From>
         <To>globalThis.mono_wasm_assembly_already_added</To>
       </BlazorProjectReplaceDotnetJsFileContents>
+      <BlazorProjectReplaceDotnetJsFileContents Include="accessing mono_set_timeout_exec">
+	      <From>this.mono_set_timeout_exec</From>
+	      <To>globalThis.mono_set_timeout_exec</To>
```

Comment:
Awesome! Just one thing, can you please align the spacing so that it matches the rest of the document?

Reply:
I changed to 4 spaces instead of tabs and auto formatted the doc, as per guidelines inCONTRIBUTING.md.

Reply:
I'm sorry @amazingalek I should update the guideline to be more specific. It is 4 spaces for C# and 2 spaces for XML and JS documents.

Reply:
No problem.

Done, 2 spaces, auto formatted.

Can I recommend tabs everywhere?

Reply:
Thank you! I personally prefer the use of spaces over tabs, which I think is also the case for other repositories as well.